core/test/financial_templates/ExpiringMultiParty.js

```diff
@@ -30,6 +31,18 @@ contract("ExpiringMultiParty", function(accounts) {
       sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
       disputerDisputeRewardPct: { rawValue: toWei("0.1") }
     };
+
+    identifierWhitelist = await IdentifierWhitelist.deployed();
+    await identifierWhitelist.addSupportedIdentifier(constructorParams.priceFeedIdentifier, {
+      from: accounts[0]
+    });
+
+    // Register the identifer white list with the finder as well
+    const identifierWhitelistName = web3.utils.utf8ToHex("IdentifierWhitelist");
+    await finder.changeImplementationAddress(identifierWhitelistName, identifierWhitelist.address, {
+      from: accounts[0]
+    });
```

Comment:
nit: the whitelist should already be registered with the finder during the migrations.